io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,411 @@
+        sha256: 56e7e64ed86d7d9ec59500fd34f26f881bdb9d541916d9a817c3bfb8cf0e9508
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --with-icu=/app --prefix=/app --libdir=/app/lib
+      - ./b2 install --prefix=/app --libdir=/app/lib -d2 -j$(nproc) threading=multi link=shared,static
+    config-opts:
+      - --without-libraries=[python, mpi]
+    build-options:
+      cxxflags: "-std=c++14"
+    cleanup:
+      - /app/share/boost
+    sources:
+      - type: archive
+        url: https://github.com/boostorg/boost/releases/download/boost-1.83.0/boost-1.83.0.tar.gz
+        sha256: 0c6049764e80aa32754acd7d4f179fd5551d8172a83b71532ae093e7384e98da
+
+  - name: boost-python3
+    buildsystem: simple
+    build-commands:
+      - echo "using python" > user-config.jam
+      - echo ": 3.12" >> user-config.jam
+      - echo ": /usr/bin/python3" >> user-config.jam
+      - echo ": /usr/include/python3.12" >> user-config.jam
+      - echo ": /usr/lib" >> user-config.jam
+      - echo ";" >> user-config.jam
+      - ./bootstrap.sh --prefix=/app --libdir=/app/lib --with-python=python3.12 --with-python-root=/usr --with-libraries=python
+      - ./b2 --build-dir=build-python3 --stagedir=stage-python3 --prefix=/app --libdir=/app/lib abi=sysv linkflags=-L/usr/lib --user-config=user-config.jam python=3.12 -d2 -j$(nproc) --layout=system threading=multi link=shared,static install
+    build-options:
+      cxxflags: "-std=c++14"
+      env:
+        - PYTHON: "/usr/bin/python3"
+        - PYTHONINCLUDE: "/usr/include/python3.12"
+        - PYTHONLIB: "/usr/lib"
+    cleanup:
+      - /app/share/boost
+    sources:
+      - type: archive
+        url: https://github.com/boostorg/boost/releases/download/boost-1.83.0/boost-1.83.0.tar.gz
+        sha256: 0c6049764e80aa32754acd7d4f179fd5551d8172a83b71532ae093e7384e98da
+
+  - name: libdwarf
+    buildsystem: autotools
+    config-opts:
+      - --enable-shared
+    build-options:
+      env:
+        PKG_CONFIG_PATH: /app/lib/pkgconfig
+    sources:
+      - type: archive
+        url: https://www.prevanders.net/libdwarf-0.11.0.tar.xz
+        sha256: 846071fb220ac1952f9f15ebbac6c7831ef50d0369b772c07a8a8139a42e07d2
+
+  - name: gsl
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/gsl/gsl-2.8.tar.gz
+        sha256: 6a99eeed15632c6354895b1dd542ed5a855c0f15d9ad1326c6fe2b2c9e423190
+
+  - name: glm
+    buildsystem: cmake
+    config-opts:
+     - -DGLM_BUILD_TESTS=OFF
+     - -DBUILD_SHARED_LIBS=ON
+     - -DCMAKE_INSTALL_PREFIX=/app
+    sources:
+      - type: archive
+        url: https://github.com/g-truc/glm/archive/refs/tags/1.0.1.tar.gz
+        sha256: 9f3174561fd26904b23f0db5e560971cbf9b3cbda0b280f04d5c379d03bf234c
+
+  - name: openblas
+    buildsystem: simple
+    builddir: true
+    build-commands:
+      - export DYNAMIC_ARCH=1
```

Comment:
this is not how you build this, all of these should be set via config-opts, see https://github.com/flathub/io.github.hydrusnetwork.hydrus/blob/3952f61e383b764aa5be580cb8d531a496d7fa78/io.github.hydrusnetwork.hydrus.yml#L209-L224